libraries/lib-datahandler/src/main/java/org/eclipse/sw360/datahandler/permissions/DocumentPermissions.java

```diff
@@ -78,6 +79,8 @@ protected boolean getStandardPermissions(RequestedAction action) {
             case USERS:
             case CLEARING:
                 return PermissionUtils.isAdmin(user) || isModerator();
+            case WRITE_ECC:
+                return PermissionUtils.isAdmin(user);
```

Comment:
**I am OK with the solution given by Alex,** although I think that my change would make it more future proof and more consisted. I prefer to place the logic in the simplest possible location, and move it out later once necessary.